deploy/helm/sumologic/conf/metrics/otelcol/exporters.yaml

```diff
@@ -0,0 +1,106 @@
+sumologic/default:
+  metric_format: prometheus
+  endpoint: ${SUMO_ENDPOINT_DEFAULT_METRICS_SOURCE}
+  sending_queue:
+    enabled: true
+{{- if .Values.metadata.persistence.enabled }}
+    storage: file_storage
+{{- end }}
+    num_consumers: 10
+    ## setting queue_size a high number, so we always use maximum space of the storage
+    ## minimal alert non-triggering queue size (if only one exporter is being used): 10GB/16MB = 640
+    queue_size: 10_000
+  max_request_body_size: 16_777_216  # 16 MB before compression
+  ## set timeout to 30s due to big requests
+  timeout: 30s
+sumologic/apiserver:
+  metric_format: prometheus
```

Comment:
Is the otel metrics collector going to share the exporter configuration with otel metadata?